pocs/tests/utils/test_image_utils.py

```diff
@@ -21,6 +39,49 @@ def test_crop_data():
     assert cropped03.sum() == 36.
 
 
+def test_make_pretty_image(solved_fits_file, tiny_fits_file, save_environ):
+    # First make a dir and put the files in it
+    with tempfile.TemporaryDirectory() as tmpdir:
+        fz_file = os.path.join(tmpdir, os.path.basename(solved_fits_file))
+        fits_file = os.path.join(tmpdir, os.path.basename(tiny_fits_file))
+        # TODO Add a small CR2 file to our sample image files.
```

Comment:
I recall that it was actually really hard to find a decently small CR2 file, which is why one was never placed there. There isn't really a tool to crop CR2 that still produces as CR2 as a result. Could maybe dig around a bit more to find one.

Reply:
Agreed, I wasn't able to find a cropped CR2 file. There are some that are nearer to 5MB than 20MB available, but not small test images. What I found surprising was that github had very few such files, and the programs that I found that can read CR2 files (e.g. libraw) apparently don't have publicly visible tests as they don't have publicly visible sample data to test against.